addon/components/polaris-date-picker.js

```diff
@@ -316,4 +353,14 @@ export default Component.extend({
       this.set('focusDate', null);
     },
   },
+
+  /**
+   * First day of week. Sunday by default
+   *
+   * @property weekStartsOn
+   * @public
+   * @type {String}
+   * @default 'Sunday'
+   */
+  weekStartsOn: weekdays.Sunday,
```

Comment:
This guy should probs be up with the rest of the public interface 😉

Reply:
@andrewpye Ah meant to comment on this, the linter hates these types of properties (it considers this an "unknown property type") and wants everything above it. I tried ignoring with eslint and prettier comments but it didn't seem to work. Any ideas on working around this?

Reply:
Unfortunately I don't, but maybe @sivakumar-kailasam does?

Reply:
Looks like assigning to a variable first fixes it. Kinda wack, but better than having it at the bottom all willy nilly (I think)